### PR TITLE
chore: add link in machine learning for krish naik

### PR DESCRIPTION
**Pull request by suzy-g38**
## Screenshots

image

### CODE REVIEW

File: database/youtube/machine-learning.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
     "description": "3Blue1Brown, by Grant Sanderson, is some combination of math and entertainment, depending on your disposition. The goal is for explanations to be driven by animations and for difficult problems to be made simple with changes in perspective.",
     "url": "https://www.youtube.com/@3blue1brown",
     "category": "youtube",
     "subcategory": "machine-learning",
     "language": "english"
-}]
+  },
+  {
+    "name": "Krish Naik",
+    "description": "I am the cofounder of iNeuron and my experience is pioneering in machine learning, deep learning, and computer vision,an educator, and a mentor, with over 10 years' experience in the industry. This is my YouTube channel where I explain various topics on machine learning, deep learning, and AI with many real-world problem scenarios. I have delivered over 30 tech talks on data science, machine learning, and AI at various meet-ups, technical institutions, and community-arranged forums. My main aim is to make everyone familiar of ML and AI.Please subscribe and support the channel. As i love new technology, all these videos are free and I promise to make more interesting content as we go ahead.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:12]:
```suggestion
    "description": "I am the cofounder of iNeuron and my experience is pioneering machine learning, deep learning, and computer vision. I am also an educator, and a mentor, with over 10 years' experience in the industry. This is my YouTube channel where I explain various topics on machine learning, deep learning, and AI with many real-world problem scenarios. I have delivered over 30 tech talks on data science, machine learning, and AI at various meet-ups, technical institutions, and community-arranged forums. My main aim is to make everyone familiar of ML and AI. Please subscribe and support the channel. As a person with a passion for making tech education accessible, all these videos are free. I promise to make more interesting content as we go ahead.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @suzy-g38! Thanks for sharing your site with us! :) I just made some suggestions to improve your description! :) Also, you have some branches to resolve, so work on those as well. 😊

**Review by Harsh-2909 — COMMENTED** [review]
Would like to ask everyone regarding this opinion but I think wouldn't it be better if we write the description in a 3rd persons perspective instead of 1st person .

File: database/youtube/machine-learning.json (hunk 1 above)
Comment (rupali-codes) [reply database/youtube/machine-learning.json:12]: We would need to update the description because it's too large

**Comment by suzy-g38** [comment]
> Hi @suzy-g38! Thanks for sharing your site with us! :) I just made some suggestions to improve your description! :) Also, you have some branches to resolve, so work on those as well. 😊

I actually copy pasted the description or about that Krish Naik mentioned on his youtube channel. I actually found that the description of another channel 3Blue1Brown is also copy-pasted from their youtube channel's description.
Okay, then I will make the changes.

**Review by suzy-g38 — COMMENTED** [review]
I have changed the description of Krish Naik

**Comment by suzy-g38** [comment]
> Hi @suzy-g38! Thanks for sharing your site with us! :) I just made some suggestions to improve your description! :) Also, you have some branches to resolve, so work on those as well. 😊

I have updated the necessary changes that were asked for.

**Comment by rupali-codes** [comment]
Could you resolve the conflicts?

**Comment by suzy-g38** [comment]
> Could you resolve the conflicts?

Sorry I didn't notice.....yess ofc.

**Comment by suzy-g38** [comment]
> Could you resolve the conflicts?

I did it please review it.

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by vikasganiga05 — APPROVED** [review]
Your contribution is highly appreciated and it looks fantastic! :tada:
Thank you so much for your efforts.

**Review by CBID2 — APPROVED** [review]
Looks great @suzy-g38! :) LGTM! :)

**Pull request merged**